xbmc/Application.h

```diff
@@ -357,7 +357,7 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
   ReplayGainSettings& GetReplayGainSettings() { return m_replayGainSettings; }
 
   void SetLoggingIn(bool switchingProfiles);
-  
+
```

Comment:
please drop all cosmetics from the PR. this PR has very critical changes that should not be mixed with cosmetics

Reply:
These were cleaned up automatically from Eclipse. I'll disable that particular feature and try to clean up the commit.